.yamato/upm-ci.yml

```diff
@@ -53,6 +79,35 @@ test_{{ platform.name }}_{{ editor.version }}:
 {% endfor %}
 {% endfor %}
 
+{% for editor in ios_test_editors %}
+test_iOS_{{ editor.version }}:
+  name: Test {{ editor.version }} on iOS
+  agent:
+    type: Unity::mobile::iPhone
+    image: mobile/macos-10.13-testing:stable
+    flavor: b1.medium
+  commands:
+    - brew tap --force-auto-update unity/unity git@github.cds.internal.unity3d.com:unity/homebrew-unity.git
+    - brew install unity-config
+    - unity-config settings editor-path .Editor
+    - unity-config project create .MyTestProject
+{%- for dependency in mobile_dependencies -%}
```

Comment:
You're creating project from scratch, wouldn't it be easier for you to have a commited Unity project?

Reply:
The committed project would have to be maintained for the oldest Unity version supported by package and any later versions become prone to any issues that may arise when upgrading project, while this creates a clean Unity project for given version.